.github/workflows/common-release.yml

```diff
@@ -83,3 +83,32 @@ jobs:
             export GIT_COMMITTER_DATE="$(git log -1 --format=%aD ${{ inputs.versionPrefix }}${SEMVER_VERSION})"
             git push origin "${{ inputs.versionPrefix }}${SEMVER_VERSION}" -f
           fi
+
+      - name: Create Issue on Failed workflow
+        if: ${{ failure() }}
+        run: |
+          if curl -sL --fail\
+              -X POST \
+              -H "Accept: application/vnd.github.v3+json" \
+              -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
+              https://api.github.com/repos/${{ github.repository }}/issues \
+              -d '{"title":"Release Pipeline failed","body":"For details see: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}","labels":["bug"]}' \
+              -o /dev/null; then
+              echo "Issue successfully created"
+          else
+              echo "Issue creation failed"
+          fi
+
+      - name: Create Slack Message on Failed Workflow
+        if: ${{ failure() }}
+        run: |
+          notification_text=":ibm-warning-filled: Release Pipeline failed. For details see: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}."
```

Comment:
Can we have the alert link to the GIT issue? you probably need to move this code into the same step as the issue creation to be able to get the issue number from the api response. Perhaps just call the step "Failure alert" or something